client/me/get-apps/mobile-download-card.jsx

```diff
@@ -234,6 +254,7 @@ class MobileDownloadCard extends React.Component {
 
 	onChange = phoneNumber => {
 		this.setState( {
+			...this.state,
```

Comment:
is this necessary? `setState` already merges the current state with the passed value.

Reply:
TIL! 

Adjusted in 8ac8028